Gemfile.lock

```diff
@@ -2,7 +2,7 @@ PATH
   remote: .
   specs:
     noticed (1.5.7)
-      http (>= 4.0.0)
+      faraday (>= 0.17.3, < 2.0)
```

Comment:
To provide compatibility with `googleauth (1.1.0)` which also depends on `faraday (>= 0.17.3, < 2.0)`
I suppose faraday can be bumped to 2.1 with googleauth but I think it  should be done in separate request